example/src/test/java/com/capgemini/mythaistar/stepdefs/bookTable/BookTableSteps.java

```diff
@@ -0,0 +1,5 @@
+package com.capgemini.mythaistar.stepdefs.bookTable;
```

Comment:
Please don't push empty class to develop. Once it has content will be merged. Now please remove this class from pull request